velox/expression/tests/CastExprTest.cpp

```diff
@@ -224,6 +224,29 @@ TEST_F(CastExprTest, timestamp) {
       });
 }
 
+TEST_F(CastExprTest, timestampToString) {
```

Comment:
nit: unrelated, but it would be nice to rename existing 'timestamp' test to 'stringToTimestamp' for consistency.